resources/js/components/attribute/Tabular.vue

```diff
@@ -117,6 +117,15 @@
                             @change="e => updateDirtyState(e, $index, column.id)"
                         />
 
+                        <date-attribute
```

Comment:
Should habe been <date-range-attribute

Reply:
Fixed in [d9d0bb2](https://github.com/DH-Center-Tuebingen/Spacialist/pull/479/commits/d9d0bb2e9ca5ba5b8f32282ca8e97cf309821f47)